### ISSUE TITLE
CSV export of submissions with repeaters don't correctly match data to fields/columns

### ISSUE THREAD

**Issue by simple-nicholas** [verbb/formie#479]
**Description**
CSV Export's of form submissions with a repeater part way through the form don't account for the repeater correctly, and some rows match data to the wrong column. Tricky to explain, easy to see in a spreadsheet.

**Steps to reproduce**
1. Create a form with a repeater part way through
2. Create test submissions with varying numbers of repeater lines
3. Export test submissions as CSV
4. Check test submissions against resulting CSV

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

(I don't believe any of these affect the outcome)

**Additional info**
- Plugin version: 1.4.5
- Craft version: 3.6.15
- Multi-site: No

**Additional context**
I recreated the issue on a basic/empty site as an example

[elements.csv](https://github.com/verbb/formie/files/6572730/elements.csv)

<img width="235" alt="Screen Shot 2021-06-01 at 10 39 35 am" src="https://user-images.githubusercontent.com/50125070/120253309-2d0be700-c2c6-11eb-869f-037a5bd49838.png"><img width="250" alt="Screen Shot 2021-06-01 at 10 39 31 am" src="https://user-images.githubusercontent.com/50125070/120253304-2c735080-c2c6-11eb-8ff5-572ff83f9e22.png"><img width="215" alt="Screen Shot 2021-06-01 at 10 39 27 am" src="https://user-images.githubusercontent.com/50125070/120253299-29786000-c2c6-11eb-8729-9a04b42bf8ef.png">

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by simple-nicholas** [comment]
That appears to be working, thank you for the prompt fix!

**Comment by engram-design** [comment]
Fixed in [1.4.6](https://github.com/verbb/formie/releases/tag/1.4.6)
